Automattic-Tracks-iOS/Model/Core Data/TracksContextManager.h

```diff
@@ -3,8 +3,22 @@
 
 @interface TracksContextManager : NSObject
 
-@property (nonatomic, strong) NSManagedObjectContext *managedObjectContext;
-@property (nonatomic, strong) NSManagedObjectModel *managedObjectModel;
-@property (nonatomic, strong) NSPersistentStoreCoordinator *persistentStoreCoordinator;
+@property (nonnull, nonatomic, strong) NSManagedObjectContext *managedObjectContext;
+@property (nonnull, nonatomic, strong) NSManagedObjectModel *managedObjectModel;
+@property (nonnull, nonatomic, strong) NSPersistentStoreCoordinator *persistentStoreCoordinator;
```

Comment:
I had to add the annotation here too to avoid warnings. It makes the diff a bit noisier but it's good to have it anyways.

The implementation of all those getters makes sure they are not `nil`, or rather, fails if they are, so `nonnullable` seemed appropriate.

Reply:
Yup that's the perfect approach Gio!